finetune/target_models/masked_language_model.py

```diff
@@ -12,6 +12,43 @@
 from tensorflow.data import Dataset
 
 
+def get_mask(seq_len, config):
+    mask_proba = config.mask_proba / config.mask_spans
+
+    a = np.random.rand(seq_len)
+    if config.table_mask_bias:
+        # init_mlm_mask = a < np.flip(np.arange(0, mask_proba * 2, mask_proba * 2/seq_len))
+        init_mlm_mask = a < [.5] * 20 + [mask_proba / 2] * (seq_len - 20)
+    else:
+        init_mlm_mask = a < mask_proba
+
+    if config.mask_spans > 1:
```

Comment:
This block dilates the masks to the right by mask_spans? So if you have [True, False, False] and mask_spans = 1 you get [True, True, False] ?

Reply:
correct